src/components/monitor/monitor.jsx

```diff
@@ -52,6 +52,7 @@ const MonitorComponent = props => (
             defaultClassNameDragging={styles.dragging}
             disabled={!props.draggable}
             onStop={props.onDragEnd}
+            scale={props.scale}
```

Comment:
drilling this scale prop to the draggable element corrects the drag deltas for different stage sizes